test/test_server.c

```diff
@@ -0,0 +1,998 @@
+ /*
+ * Copyright (c) 2015-2018 Intel, Inc.  All rights reserved.
```

Comment:
As this is a new file - only Mellanox copyright should be listed for now.

Reply:
This file got the code moved from `test/utils.c` file, so I kept copyrights.